fast64_internal/sm64/sm64_objects.py

```diff
@@ -1507,14 +1501,22 @@ def upgrade_object(obj):
 			del obj["sm64_obj_bparam"]
 
 		# get combined bparams, if they arent the default value then return because they have been set
-		game_object.combined_bparams = game_object.get_combined_bparams()
-		if game_object.combined_bparams != DEFAULT_BEHAVIOR_PARAMS:
+		combined_bparams = game_object.get_combined_bparams()
+		if combined_bparams != DEFAULT_BEHAVIOR_PARAMS:
 			return
 
 		# If bparams arent the default bparams, disable `use_individual_params`
 		if (game_object.bparams != DEFAULT_BEHAVIOR_PARAMS):
 			game_object.use_individual_params = False
 
+	@staticmethod
+	def remove_unused_props(obj: bpy.types.Object):
+		game_object: SM64_GameObjectProperties = obj.fast64.sm64.game_object
+
+  		# replaced "combined_bparams" with method `get_combined_bparams`
+		if "combined_bparams" in game_object:
+			del game_object["combined_bparams"]
```

Comment:
imo there is no need to worry about such cleanup. code for upgrading is already a burden to maintain, I don't think we need code dealing with removed props

I think simpler upgrade code is worth the slight blend file bloat, thoughts?

Reply:
yeah thats a good point! dropping it